modules/ROOT/pages/using-bitronix-to-manage-transactions.adoc

```diff
@@ -0,0 +1,142 @@
+= Using Bitronix to Manage Transactions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: bitronix, transaction, xa, management, bti
+
+[WARNING]
+====
+If the Mule server crashes or loses connectivity to a resource manager (such as a JMS broker, database, etc.), this can cause a transaction to fail abnormally between the prepare phase and the commit phase of the two-phase commit protocol (2PC). In this case, the transaction manager needs to run a recovery process to complete any in-flight transaction that was taking place during the failure. While this kind of failure is unlikely, a mechanism is needed to retain data consistency in the event of this happening.
+
+Bitronix allows Mule Runtime to automatically recover interrupted transactions on restart.
+====
+
+
+== Assumptions
+
+This document assumes you are familiar with working with xref:shared-resources.adoc[Shared Resources] in Mule, and xref:transaction-management.adoc[Transaction Management].
```

Comment:
Why should a user have to understand shared resources to understand bitronix transaction manager?

Reply:
Domains is explained in shared resources doc